### ISSUE TITLE
android does not show passcode

### ISSUE THREAD

**Issue by matwming** [naoufal/react-native-touch-id#232]
This package works great on ios with touch ID, face ID and passcode.
On android, the touch ID also works if touch ID is enabled.
BUT, when touch ID on android is not enabled, unlike ios, it will not give users an option to enter passcode. **It throws "Touch ID Error"**.
**Is it possible to show a passcode on android when touch ID is not enabled?**
Many thanks.
React Native:0.59.8

**Comment by galimpian** [comment]
I agree, I'm having a similar issue, all I want to do is just detect if the users android device has touchID for changing out text.

**Comment by matwming** [comment]
Hi Galimpian,
I think you can easily detect whether users enable touchID on android.
If a user does not enable touch ID, this package will throw an error, you just need to catch the error and change the error message to string using toString(), and check if it includes 'Touch ID Error' by includes(), and you can do anything you like such as changing text. 
`.catch(err=>{
if(err.toString().includes('Touch ID Error')){
// do anything you like here
}
})`
Do not forget use Platform api from react native to to make sure the user is actually on android platform
